### ISSUE TITLE
Add grafana dashboard

### ISSUE THREAD

**Issue by aebruno** [ubccr/slurm-exporter#12]
Just adding an issue for this. We should include a grafana dashboard for use with slurm-exporter.

**Comment by prime-hacker** [comment]
Hello, I think this [Slurm Dashboard](https://grafana.com/grafana/dashboards/4323-slurm-dashboard/) has been created a while ago. Is it sufficient for this exporter or you mean another dashboard?

**Comment by aebruno** [comment]
@prime-hacker Yes that dashboard will work for some metrics but is missing others. We built one specific for our exporter based of that one which includes some additional metrics.

**Comment by prime-hacker** [comment]
Cool. Can you give me a link to this dashboard if it is published?

**Comment by aebruno** [comment]
Yes, for sure. we'll update this issue once we publish it.
